builtin/providers/vsphere/resource_vsphere_virtual_machine_test.go

```diff
@@ -455,6 +455,168 @@ func TestAccVSphereVirtualMachine_createWithCdrom(t *testing.T) {
 	})
 }
 
+func TestAccVSphereVirtualMachine_updateMemory(t *testing.T) {
+	var vm virtualMachine
+	var locationOpt string
+	var datastoreOpt string
+
+	if v := os.Getenv("VSPHERE_DATACENTER"); v != "" {
```

Comment:
how would datacenter usually be sent to the VM? Would it be created and then passed as a resource?